DataConnectors/Aruba ClearPass/Connector_Syslog_ArubaClearPass.json

```diff
@@ -0,0 +1,94 @@
+{
+    "id": "ArubaClearPass",
+    "title": "Aruba ClearPass",
+    "publisher": "Aruba Networks",
+    "descriptionMarkdown": "The [Aruba ClearPass](https://www.arubanetworks.com/products/security/network-access-control/secure-access/) connector allows you to easily connect your Aruba ClearPass with Azure Sentinel, to create custom dashboards, alerts, and improve investigation. This gives you more insight into your organization’s network and improves your security operation capabilities.",
+    "additionalRequirementBanner":"These queries are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **ArubaClearPass** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinel-arubaclearpass-parser)",
+    "graphQueries": [{
+        "metricName": "Total data received",
+        "legend": "Aruba ClearPass",
+        "baseQuery": "CommonSecurityLog​ \n| where DeviceVendor == \"Aruba Networks\" and DeviceProduct == \"ClearPass\""
+    }],
+    "sampleQueries": [{
+        "description": "Top 10 Events by Username",
+        "query": "ArubaClearPass \n | summarize count() by UserName \n| top 10 by count_"
+    }, {
+        "description": "Top 10 Error Codes",
+        "query": "ArubaClearPass \n | summarize count() by ErrorCode \n| top 10 by count_"
+    }],
+    "connectivityCriterias": [{
+        "type": "IsConnectedQuery",
+        "value": ["CommonSecurityLog​ \n| where DeviceVendor == \"Aruba Networks\" and DeviceProduct == \"ClearPass\"\n | summarize LastLogReceived = max(TimeGenerated)\n | project IsConnected = LastLogReceived > ago(30d)"]
```

Comment:
Please remove [ at the beginning of the query and ] at the end.

Reply:
brackets removed